doc/api/process.md

```diff
@@ -1336,6 +1336,45 @@ In custom builds from non-release versions of the source tree, only the
 `name` property may be present. The additional properties should not be
 relied upon to exist.
 
+## process.resourceUsage()
+<!-- YAML
+added: REPLACEME
+-->
+
+* Returns: {Object} containing resource usage measures for the current process
+
+Note: Windows only provides the following resource usage measures:
+
+- `userCpuTimeUsedSec`
+- `userCpuTimeUsedMs`
+- `systemCpuTimeUsedSec`
+- `systemCpuTimeUsedMs`
```

Comment:
https://github.com/libuv/libuv/issues/342 and https://github.com/libuv/libuv/pull/1041 added block input/output operations for Windows.